dataset/dbengine.py

```diff
@@ -1,15 +1,17 @@
+from functools import partial
+from multiprocessing import Pool
+from string import Template
 import logging
```

Comment:
No need to address during this PR but we should ordering imports alphabetically.